kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -117,26 +190,6 @@ define([
                 },
             });
 
-            /*
-             * Or in fact any parameter value at any time...
-             */
-            fieldWidget.bus.on('get-parameter-value', (message) => {
```

Comment:
AFAICT, this message isn't used by any widget, though all the high level widget controllers listen for it.

Reply:
Got to get those LOC numbers up somehow!